### ISSUE TITLE
Nodes are not revealing in details panel.

### ISSUE THREAD

**Issue by dreamflyer** [mulesoft/api-workbench#483]
1. paste following content into the editor:

```raml
#%RAML 1.0 
title: Pet shop
version: 1
baseUri: /shop
 
types:
  Pet:
    properties:
      name: string
      kind: string
      price: number
    example:
      name: "Snoopy"
      kind: "Mammal"
      price: 100
 
/pets:
  get:
    responses:
      200:
        body:
          application/json:
  post:
    body:
      application/json:
        type: Pet
  /{id}:
    put:
      body:
        application/json:
    delete:
      responses:
        204:
```
2. put cursor into "application/json" under "post" method.

expected:
![](http://apiworkbench.com/docs/images/GettingStarted_Outline.png)
actually:
![screen shot 2017-05-10 at 3 12 05 pm](https://cloud.githubusercontent.com/assets/712787/25891494/7a61b4ce-359b-11e7-81b6-7f31b054f03f.png)

also "Resources" tab is wrongly named in tutorial.

**Comment by dreamflyer** [comment]
related to #476.

**Comment by dreamflyer** [comment]
will be published in v0.8.46.
